.github/workflows/Dockerfile

```
@@ -1,14 +1,18 @@
-FROM ghcr.io/precice/precice:latest
+ARG PRECICE_TAG=latest
+ARG from=precice/precice:${PRECICE_TAG}
+FROM $from
 
+ARG OPENFOAM_EXECUTABLE=openfoam2112
+ARG OPENFOAM_ADAPTER_REF=develop
 # Install OpenFOAM
-ENV OPENFOAM_VERSION=2112
-RUN wget -q -O - https://dl.openfoam.com/add-debian-repo.sh | sudo bash
-RUN sudo apt-get -qq install openfoam${OPENFOAM_VERSION}-dev
+RUN apt-get update
+RUN wget -q -O - https://dl.openfoam.com/add-debian-repo.sh | bash
+RUN apt-get -qq install $OPENFOAM_EXECUTABLE-dev git
```

Shouldn't this be in this order?

```suggestion
RUN wget -q -O - https://dl.openfoam.com/add-debian-repo.sh | bash
RUN apt-get update
RUN apt-get -qq install $OPENFOAM_EXECUTABLE-dev git
```

---

No i think we need to update the package lists before we can execute the script. The script does not make sure it has the newest package lists and sometimes the apt will fail because package mirrors moved.